### PR TITLE
Fix metadata.json

### PR DESCRIPTION
**Pull request by mcanevet**

### CODE REVIEW

File: metadata.json [hunk 1]
```diff
@@ -1,11 +1,11 @@
 {
-  "name": "puppetlabs-nodejs",
+  "name": "puppet-nodejs",
   "version": "0.8.0",
   "author": "puppetlabs",
```
Comment (nibalizer) [inline metadata.json]: This needs to be puppetcommunity.

File: metadata.json [hunk 2]
```diff
@@ -1,11 +1,11 @@
 {
-  "name": "puppetlabs-nodejs",
+  "name": "puppetcommunity-nodejs",
```
Comment (nibalizer) [inline metadata.json]: This should be puppet.

File: metadata.json [hunk 3]
```diff
@@ -1,11 +1,11 @@
...
   "version": "0.8.0",
   "author": "puppetlabs",
```
Comment (nibalizer) [inline metadata.json]: This should be puppetcommunity

File: metadata.json [hunk 4]
```diff
@@ -1,11 +1,11 @@
...
   "summary": "Install Node.js package and npm package provider.",
   "license": "Apache-2.0",
-  "source": "https://github.com/puppetlabs/puppetlabs-nodejs",
-  "project_page": "https://github.com/puppetlabs/puppetlabs-nodejs",
+  "source": "https://github.com/puppet-community/puppet-nodejs",
+  "project_page": "https://github.com/puppet-community/puppet-nodejs",
   "issues_url": "https://tickets.puppetlabs.com/browse/MODULES",
```
Comment (nibalizer) [inline metadata.json]: This is a bit uncertain. Should we continue to use puppetlabs MODULES for this or revert to using github issues?

**Comment by nibalizer** [comment]
Sorry. I had the particulars of this wrong.

File: metadata.json (hunk 4 above)
Comment (mcanevet) [inline metadata.json]: @igalic @daenney should we enable issues on puppetcommunity modules?

**Comment by daenney** [comment]
If its now "our" module we should sue Github issues and not expect everyone to create a Puppetlabs account and track the issues there. Also makes it easier to link PRs to issues etc.

**Comment by mcanevet** [comment]
@daenney cool you allow github issues on this module? Or give me the permissions to do it?

**Comment by mcanevet** [comment]
@daenney issues before migration to Jira appeared...

**Comment by mcanevet** [comment]
@daenney @nibalizer can we merge this and make a first release in the `puppet` namespace?

File: metadata.json [hunk 5]
```diff
@@ -1,12 +1,12 @@
 {
-  "name": "puppetlabs-nodejs",
+  "name": "puppet-nodejs",
   "version": "0.8.0",
```
Comment (juniorsysadmin) [inline metadata.json]: Should the version number be reset?

Comment (mcanevet) [inline metadata.json]: @juniorsysadmin yes, but maybe after merging this PR, juste before releasing a new version

Comment (nibalizer) [inline metadata.json]: We can release a 0.9.0, no need to reset.

File: metadata.json [hunk 6]
```diff
@@ -1,12 +1,12 @@
...
   "author": "puppetlabs",
```
Comment (nibalizer) [inline metadata.json]: This should be puppetcommunity.

Comment (mcanevet) [inline metadata.json]: @nibalizer the namespace on the forge is `puppet`, not `puppetcommunity`, so I think `puppet` is fine

**Comment by juniorsysadmin** [comment]
Don't forget #130 before the release.

File: metadata.json [hunk 7]
```diff
@@ -1,12 +1,12 @@
 {
-  "name": "puppetlabs-nodejs",
-  "version": "0.8.0",
-  "author": "puppetlabs",
+  "name": "puppet-nodejs",
+  "version": "0.9.0",
```
Comment (igalic) [inline metadata.json]: i'd suggest we give it a 1.0.0

**Pull request merged**